.github/workflows/release.yml

```diff
@@ -169,3 +169,26 @@ jobs:
           asset_path: ./synth.exe
           asset_name: synth-windows-latest-x86_64.exe
           asset_content_type: application/octet-stream
+      - id: install-wix
+        run: nuget install WiX -Version 3.11.2
+      - id: install-cargo-wix
+          cargo +nightly install cargo-wix
+      - id: copy-target
+        run: |
+          mkdir synth/target
+          mkdir synth/target/release
+          copy target/release/synth.exe synth/target/release
+      - id: wix
+        run: |
+          cd synth
+          cargo wix
```

Comment:
The `main.wxs` is empty, which I suppose means everything is left to default?

When I did this locally, that file was populated with boilerplate and in particular the installer offered to add the binary to the path. Will that be done with defaults as well?

Reply:
Yes -- At least that was what worked for me.